tests/app/clients/test_twilio.py

```diff
@@ -188,13 +236,166 @@ def test_send_sms_raises_if_twilio_rejects(notify_api, mocker):
 
 
 def test_send_sms_raises_if_twilio_fails_to_return_json(notify_api, mocker):
-    to = '+61412345678'
-    content = 'my message'
-    reference = 'my reference'
+    to = "+61412345678"
+    content = "my message"
+    reference = "my reference"
 
-    response_dict = 'not JSON'
+    response_dict = "not JSON"
 
     with pytest.raises(ValueError), requests_mock.Mocker() as request_mock:
-        request_mock.post('https://api.twilio.com/2010-04-01/Accounts/TWILIO_TEST_ACCOUNT_SID_XXX/Messages.json',
-                          text=response_dict, status_code=200)
+        request_mock.post(
+            "https://api.twilio.com/2010-04-01/Accounts/TWILIO_TEST_ACCOUNT_SID_XXX/Messages.json",
+            text=response_dict,
+            status_code=200,
+        )
         twilio_sms_client.send_sms(to, content, reference)
+
+
+@pytest.mark.parametrize(
+    "environment, expected_prefix",
+    [
+        ("staging", "staging-"),
+        ("performance", "sandbox-"),
+        ("production", ""),
+        ("development", "dev-"),
+    ],
```

Comment:
You use this list in 3 tests.  I would set it as a constant at the top of the test file so you don't repeat yourself, and then:
```
@pytest.mark.parametrize("environment, expected_prefix", ENV_LIST)
```